plugins/com.aptana.explorer/src/com/aptana/explorer/internal/ui/SingleProjectView.java

```diff
@@ -508,7 +511,13 @@ public ISelection getSelection()
 			{
 				ISelection sel = super.getSelection();
 				if (sel.isEmpty() && selectedProject != null)
-					return new StructuredSelection(selectedProject);
+				{
+					ArrayList<Object> selectedProjectList = new ArrayList<Object>();
```

Comment:
**Two issues:**
1. Assign to general List
2. Use specific object type IProject in the list

` List< IProject> selectedProjectList =  new ArrayList<IProject>;`

Reply:
Done!